src/components/themed/WalletListSortable.tsx

```diff
@@ -23,39 +20,46 @@ interface Props {
  */
 export function WalletListSortable(props: Props) {
   const { insetStyle } = props
+
   // Subscribe to account state:
   const account = useSelector(state => state.core.account)
   const currencyWallets = useWatch(account, 'currencyWallets')
 
   const [walletOrder, setWalletOrder] = React.useState(account.activeWalletIds)
-  const handleDragEnd = useHandler((params: DragEndParams<string>) => setWalletOrder(params.data))
 
-  const keyExtractor = useHandler((walletId: string) => walletId)
-  const renderItem = useHandler((params: RenderItemParams<string>) => (
-    <ScaleDecorator activeScale={0.9}>
-      <WalletListSortableRow wallet={currencyWallets[params.item]} onDrag={params.drag} />
-    </ScaleDecorator>
-  ))
-  const handleScroll = useSceneScrollHandler()
+  const handleReorder = useHandler(({ from, to }: { from: number; to: number }) => {
+    // Reorder the walletOrder array
+    const newOrder = [...walletOrder]
+    newOrder.splice(to, 0, newOrder.splice(from, 1)[0])
+    setWalletOrder(newOrder)
 
-  React.useEffect(() => () => {
+    // Update the wallet sort order in the account
     const keyStates: EdgeWalletStates = {}
-    for (let i = 0; i < walletOrder.length; ++i) {
-      const walletId = walletOrder[i]
+    for (let i = 0; i < newOrder.length; ++i) {
+      const walletId = newOrder[i]
       keyStates[walletId] = { sortIndex: i }
     }
     account.changeWalletStates(keyStates).catch(error => showError(error))
   })
 
+  const keyExtractor = (item: string) => item
```

Comment:
I think this needs a useHandler or useCallback as well.

Reply:
Using a useHandler somehow broke the feature

Reply:
Figured it out, needed to use `useCallback`